ui/effect-pulsate.js

```diff
@@ -53,7 +53,7 @@ return $.effects.effect.pulsate = function( o, done ) {
 
 	elem.animate({
 		opacity: animateTo
-	}, duration, o.easing);
+	}, duration, o.easing, function(){elem.css("opacity","");});
```

Comment:
I think, this step, because CSS attr "opacity" need delete after ending animation.
see http://danyadaro.github.io/fix-jqueryui-effects/index_fix_scott.html